specify/specify7
# Keep focus on xml code and not on linter after error

Fixes #4753


### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to a form definition.
- Go to a field and modify the name to be invalid.
- Wait for the warning panel to appear 
- See that your cursor position has not been removed.
